code/style.css

```diff
@@ -1,5 +1,172 @@
-/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
```

Comment:
Really nice to see *-selector and these "neutralizing" styles. I use them too, all the time! 🌟